doc/articles/features/working-with-xaml-hot-reload.md

```diff
@@ -85,6 +86,43 @@ Mobile targets are currently using a limited version of XAML Hot Reload and do n
 
 Hot Reload is supported by Visual Studio for WinAppSDK and provides support in unpackaged deployment mode.
 
+## Hot Reload Indicator
```

Comment:
@rajamatt, @dr1rrb, @jeromelaban 
I think we should mention that the Hot Reload Indicator is not available for WinAppSDK Packaged/Unpackaged

Reply:
Agreed.